app/src/display/widgets/Kconfig

```
@@ -1,6 +1,12 @@
 
 menu "ZMK Display Widgets"
 
+config ZMK_WIDGET_LAYER_STATUS
+    bool "Widget for highest, active layer using small icons"
+    default y
+    depends on !ZMK_SPLIT_PERIPHERAL
```

With the merge of #510, this should now be *something* like:

```suggestion
    depends on !ZMK_SPLIT || ZMK_SPLIT_BLE_ROLE_CENTRAL
```

You'd need to see if that boolean logic works for this.